packages/shared/lib/core/network/constants/chronicle-urls.constant.ts

```diff
@@ -0,0 +1,10 @@
+import { NetworkId } from '../enums'
+
+export const CHRONICLE_URLS: Readonly<{ [key in NetworkId]?: string[] }> = {
+    [NetworkId.Iota]: ['https://chronicle.stardust-mainnet.iotaledger.net/'],
```

Comment:
the rotating addresses most likely come from chrysalis not stardust, shouldnt we check the historical addresses in chrysalis instead?